### ISSUE TITLE
Still Not Scanning Library, even with Version 10

### ISSUE THREAD

**Issue by lbhdan** [rmcrackan/Libation#560]
**Describe the bug**
When I try to scan my Audible Account to add new items to my Libation Library, it just tries scanning, stops responding, and crashes.

**To Reproduce**
This happens each time I open Libation and click on Scan Library.

**Expected behavior**
Libation should be finding a bunch of new title recently added to my Audible Account.

**Platform**
Windows 10

Attach your Libation log file here.
Here's the logfile from my most recent attempt:

[Log202303.log](https://github.com/rmcrackan/Libation/files/11105510/Log202303.log)

**Comment by Mbucari** [comment]
@lbhdan Can you please back up your `LibationContext.db` file, delete it, then launch Libation and try scanning?

**Comment by lbhdan** [comment]
Done. This time it scanned properly, including the new items. However, I've lost all the download info about what I had previously added to Libation. Is there any way to get it back?

**Comment by Mbucari** [comment]
@lbhdan OK, try this. 
- Restore your old LibationContext.db
- Download [this pre-release](https://github.com/rmcrackan/Libation/actions/runs/4558237896)
- Run it and see if it will let you scan.

**Comment by rmcrackan** [comment]
Hopefully I'll have the real fix live before long. It's my oldest daughter's birthday though so if I can't fix it soon, the full version will have to wait until after she's in bed.

**Comment by rmcrackan** [comment]
@Mbucari live release 10.0.2 *should* contain the rapid fixes. Please ensure this is true. Thanks for the quick response on this

**Comment by Mbucari** [comment]
> @Mbucari live release 10.0.2 _should_ contain the rapid fixes. Please ensure this is true. Thanks for the quick response on this

Thank you! I'll check in about an hour.

**Comment by rmcrackan** [comment]
Perfect. Gracias

**Comment by lbhdan** [comment]
I've installed the official update now, and that seems to be working perfectly. I had a bit of a hiccup when I tried downloading some books, and got an error message in the download log saying "Error ensuring valid state". I finally worked out that Audible apparently didn't like my VPN running while connecting with Libation (although, confusingly, it worked fine with it running just a couple of hours ago). I paused my VPN, and downloads are proceeding apace. Thanks again for your quick response and great fixes!
